internal/kubernetes/operator/project/project.go

```diff
@@ -394,41 +394,76 @@ func buildPrivateEndpoints(peProvider store.PrivateEndpointLister, projectID str
 }
 
 func buildNetworkPeering(npProvider store.PeeringConnectionLister, projectID string) ([]atlasV1.NetworkPeer, error) {
-	var result []atlasV1.NetworkPeer
-
 	// pagination not required, max 25 entries per provider can be configured via API
-	npList, err := npProvider.PeeringConnections(projectID, &atlas.ContainersListOptions{
+	npListAWS, err := npProvider.PeeringConnections(projectID, &atlas.ContainersListOptions{
 		ListOptions: atlas.ListOptions{
 			ItemsPerPage: MaxItems,
 		},
+		ProviderName: string(provider.ProviderAWS),
 	})
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("error getting network peering connections for AWS: %s", err)
```

Comment:
let's use improved error handling for go 
```suggestion
		return nil, fmt.Errorf("error getting network peering connections for AWS: %w", err)
```

Reply:
Maybe it makes sense to add [errorlint](https://github.com/polyfloyd/go-errorlint) to golangci-lint.

Reply:
go for it, for conext we started with go1.12 when I tried to enable some of those error linters there was a lot of technical debt being flagged, not sure what the state is right now as I've been trying to fix them over time